### ISSUE TITLE
SourceMap support

### ISSUE THREAD

**Issue by ash3011** [xoofx/NUglify#21]
Does this library support SourceMap? if you could you provide me small example please?

**Comment by xoofx** [comment]
For Javascript this is the same API that was provided by the original AjaxMin, you can have a look at [this example code for js](https://github.com/xoofx/NUglify/blob/00e82cfd517806af469b2cc6ad4b3fa0c9e288ef/src/NUglify.Tests/Core/SymbolsMapTests.cs),

**Comment by asprna** [comment]
Thanks xoofx. is there an example how we can generate .map file?

**Comment by xoofx** [comment]
Have you looked at the link I gave above?

**Comment by asprna** [comment]
Yeah, It was not working due to a mistake made by me. I got this working now

Thanks heaps
